app/presenters/green_lanes/assessments_presenter.rb

```diff
@@ -64,11 +84,19 @@ def extract_exemptions(category_assessments)
       category_assessments.flat_map { |ca| ca.exemptions.map(&:code) }
     end
 
-    def process_exemptions(exemptions)
-      return [] if exemptions.nil?
+    def process_exemptions(answers)
```

Comment:
chnaged here because its actually answers that are passed in as a hash with the key being the cat_assessment number

Reply:
`process_exemptions` and `process_assessment` are not meaningful method names:
you cannot deduce what they do :)